juriscraper/opinions/united_states/state/ill.py

```diff
@@ -42,20 +48,27 @@ def _process_html(self) -> None:
         """
         for row in self.html.xpath("//table[@id='ctl04_gvDecisions']/tr"):
             cells = row.xpath(".//td")
-
             # Don't parse pagination rows or headers or footers
             if len(cells) != 7 or row.xpath(".//table"):
                 continue
-
-            name = get_row_column_text(row, 1)
-            date = get_row_column_text(row, 3)
-            citation = get_row_column_text(row, 2)
             try:
                 url = get_row_column_links(row, 1)
             except IndexError:
+                logger.info(
+                    f"Opinion without URL to file: '{str(tostring(row))}'"
+                )
                 # If the opinion file's information is missing (as with
                 # links to withdrawn opinions), skip record
                 continue
+            citation = get_row_column_text(row, 2)
+            if not citation:
+                logger.info(
+                    f"Opinion without citation: '{str(tostring(row))}'"
+                )
+                # If the opinion citation is missing, skip record
+                continue
```

Comment:
we would not want to skip a row because no citation exists.

Reply:
The docket number is parsed from the citation, without the citation it's not possible to get the case's docket.
Should cases without a docket number be included?

In the original report form Sentry, https://sentry.io/organizations/freelawproject/issues/2892226059/?project=5257254
the case that triggered the error:
```
{
    date: '12/28/2021', 
    name: 'People v. Winters', 
    neutral_citation: '', 
    url: 'https://www.illinoiscourts.gov/resources/93c5361d-28fe-4bfe-ac82-8886f07a7d65/file'
}
```
was later scraped in CL without problem:
https://www.courtlistener.com/opinion/5312698/people-v-winters/?type=o&order_by=dateFiled+desc&stat_Precedential=on&stat_Non-Precedential=on&court=illappct&page=2

Which leads me to believe that this error was due a temporary connection error to the court's website, which caused that the page was not read completely.